src/Sarif.Driver/Sdk/EntryPointUtilities.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif.Driver.Sdk
+{
+    public static class EntryPointUtilities
+    {
+        internal static string[] GenerateArguments(
+            string[] args,
+            IFileSystem fileSystem,
+            IEnvironmentVariables environmentVariables)
+        {
+            List<string> expandedArguments = new List<string>();
+
+            foreach (string argument in args)
+            {
+                if (argument[0] != '@' &&
+                    (argument.Length == 1 || argument[1] != '@'))
```

Comment:
>) [](start = 63, length = 1)

Is this an escaping mechanism? Is the idea that an argument "@@something" is treated as a literal argument ("don't interpret the leading '@' as introducing a response file name")? In that case I'd expect the argument to "@@something" to be interpreted as the literal string "@something" (and there needs to be a test for this logic).